pkg/authorization/authorizer/scope/converter_test.go

```diff
@@ -0,0 +1,211 @@
+package scope
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+)
+
+func TestSpecialEvaluator(t *testing.T) {
```

Comment:
The integration tests addresses only one special token, in that case, what about others and clusterrole one?